contributions/catalog/webapps/templates/contribute/talent_details.html

```diff
@@ -20,208 +21,221 @@
+                <h4>Icon</h4>
+            </div>
+            <div class="col-12 col-md-9">
+                <div class="small-body-text">{{post.icon}}</div>
+            </div>
         </div>
-        <table>
-            {% for x in post.requiredCapabilities %}
-                <tr>
-                    <td><a href="{{ url_for('contribute.capability_details', contribution_id=x['contributionId'], id=x['capabilityId']) }}">{{x['capabilityName']}}</a></td>
-                </tr>
-            {% endfor %}
-            <!-- extra row for the separation to next field -->
-            <tr>
-                <td>&nbsp;</td>
-            </tr>
-        </table>
-    </div>
-    {% endif %}
+        {% endif %}
 
-    {% if post.requiredBuildingBlocks %}
-    <div class="row">
-        <div class="col-12 col-sm-2">
-            <h4>Required Building Blocks</h4>
-        </div>
-        <table>
-            {% for x in post.requiredBuildingBlocks %}
+        {% if post.requiredCapabilities %}
+        <div class="row">
+            <div class="col-12 col-md-3">
+                <h4>Required Capabilities</h4>
+            </div>
+            <table>
+                {% for x in post.requiredCapabilities %}
+                    <tr>
+                        <td><a href="{{ url_for('contribute.capability_details', contribution_id=x['contributionId'], id=x['capabilityId']) }}">{{x['capabilityName']}}</a></td>
+                    </tr>
+                {% endfor %}
+                <!-- extra row for the separation to next field -->
                 <tr>
-                    <td>{{x}}</td>
+                    <td>&nbsp;</td>
                 </tr>
-            {% endfor %}
-            <!-- extra row for the separation to next field -->
-            <tr>
-                <td>&nbsp;</td>
-            </tr>
-        </table>
-    </div>
-    {% endif %}
-
-    {% if post.minUserPrivacyLevel %}
-    <div class="row">
-        <div class="col-12 col-sm-2">
-            <h4>Minimum user privacy level</h4>
-        </div>
-        <p class="col-sm-9">{{post.minUserPrivacyLevel}}</p>
-    </div>
-    {% endif %}
-
-    {% if post.minEndUserRoles %}
-    <div class="row">
-        <div class="col-12 col-sm-2">
-            <h4>Minimum set of end-user roles</h4>
+            </table>
         </div>
-        <p class="col-sm-9">
-            {% for role in post.minEndUserRoles %}
-                {{role}}<br>
-            {%  endfor %}</p>
-    </div>
-    {% endif %}
+        {% endif %}
 
-    {% if reviewer %}
-        {% if post.startDate %}
+        {% if post.requiredBuildingBlocks %}
         <div class="row">
-            <div class="col-12 col-sm-2">
-                <h4>Start Date</h4>
+            <div class="col-12 col-md-3">
+                <h4>Required Building Blocks</h4>
             </div>
-            <p class="col-sm-9">{{post.startDate}}</p>
+            <table>
+                {% for x in post.requiredBuildingBlocks %}
```

Comment:
The Required Building Blocks section alignment looks off. I thought of sharing as feedback as it's related to the user interface. Thanks.

 
<img width="650" alt="image" src="https://user-images.githubusercontent.com/4306604/213084786-3ffc3cdb-def7-478a-911b-e84e3a94bd4f.png">

The rest of the changes look fine to me. Thanks.

Reply:
Added a column div to fix styling.

Ready for review.

Reply:
Thanks.